### PR TITLE
AP-2305 add token utils

### PR DESCRIPTION
**Pull request by Dosexe**
## Changes

Please describe

## Checklist

- [x] Apply one of following labels; `major`, `minor`, `patch` or `skip-release`
- [x] I've updated the documentation, or no changes were necessary
- [x] I've updated the tests, or no changes were necessary

### CODE REVIEW

File: lib/jwt-utils/tokenUtils.ts [hunk 1]
```diff
@@ -0,0 +1,46 @@
+import type { JWT } from '@fastify/jwt'
+import { AuthFailedError } from '@lokalise/node-core'
+
+import { EmptyTokenError } from '../errors/publicErrors'
+
+export function generateJwtToken(
```
Comment (kibertoad) [inline lib/jwt-utils/tokenUtils.ts:6]: you need to add it to index.ts

Comment (Dosexe) [reply lib/jwt-utils/tokenUtils.ts:6]: fixed

**Comment by Dosexe** [comment]
@kibertoad Token utils were not covered with tests properly, so it drops the coverage level.

**Comment by kibertoad** [comment]
@Dosexe let's add tests now

**Review by kibertoad — APPROVED** [review]

**Comment by Dosexe** [comment]
@kibertoad  Its unclear how to simulate situation when token is not decoded, but error is not thrown.

**Pull request merged**